packages/test/src/test-workflows.ts

```diff
@@ -229,6 +247,15 @@ function makeRespondToQueryCommand(
   };
 }
 
+function makeSetChangeMarker(myChangeId: string, deprecated: boolean): coresdk.workflow_commands.IWorkflowCommand {
```

Comment:
Shouldn't we rename the marker to match the function name? PatchMarker?

Reply:
Yep, missed this one.

Reply:
Oh, right, this is because that's still how it's called in the protos/core. I will rename it in core and then come back here.